lib/thumbs/pull_request_worker.rb

```diff
@@ -136,20 +136,21 @@ def comments
     end
 
     def bot_comments
-      comments.collect { |c| c if c[:user][:login] == ENV['GITHUB_USER'] }.compact
+      comments.collect { |c| c if ["thumbot"].include?(c[:user][:login]) }.compact
     end
 
     def contains_plus_one?(comment_body)
-      comment_body =~ /\+1/
+      debug_message "match comment_body #{comment_body}"
+     (/:\+1:/.match(comment_body) || /\+1/.match(comment_body)) ? true : false
```

Comment:
I believe this change from bors will require some "training".

Technically, this will match if `comment_body = "something that should be done before I +1 this"`, but people understand the feature as requiring `comment_body = "+1 <commit_hash>"` where `commit_hash` is either in short form 7 characters or long form 40 characters so in regex [0-9a-fA-F]{7,40} .

More technically, the commit hash +1'd MUST match the last commit hash in the PR. IMO this feature of bors took some of the reviewer's attention away from the review and placed it on communicating w/ the review tool. SO, +1 to simplifying this feature to any +1 in the comment means approval.